adam/experiment/observer.py

```diff
@@ -373,7 +379,9 @@ def render_buttons_html(self, situation: HighLevelSemanticsSituation) -> str:
             )
         if not situation.is_dynamic:
             return buttons[0]
-        return "".join(buttons)
+        if perception.during and perception.during.at_some_point:
+            return "".join(buttons)
+        return "".join(buttons[0:2])
```

Comment:
With this edit, the HTML output should always have the correct number of buttons for the scene in question. (1, 2 or 3)